ydb/library/yql/udfs/common/knn/test/canondata/test.test_CosineDistance_/results.txt

```diff
@@ -149,5 +149,807 @@
                 ]
             }
         ]
+    };
+    {
+        "Write" = [
+            {
+                "Type" = [
+                    "ListType";
+                    [
+                        "StructType";
+                        [
+                            [
+                                "column0";
+                                [
+                                    "OptionalType";
+                                    [
+                                        "DataType";
+                                        "Float"
+                                    ]
+                                ]
+                            ]
+                        ]
+                    ]
+                ];
+                "Data" = [
+                    [
+                        [
+                            "0.025368094"
```

Comment:
I just wanted to be sure that you check all the results with calculator. 
I did it with all the previous results.

Reply:
I didn't checked CosineDistance/Similarity, but I checked InnerProduct, so I think it should be also correct (because we know internally it uses same function)

Reply:
Please, check one or two CosineDistance also. Just to be sure everything is ok.

Reply:
[Done](https://www.wolframalpha.com/input?i=CosineDistance%5B%7B1%2C+2%2C+3%7D%2C+%7B4%2C+5%2C+6%7D%5D%3B+1+-+CosineDistance%5B%7B1%2C+2%2C+3%7D%2C+%7B4%2C+5%2C+6%7D%5D)